src/Tizen.NUI/src/public/Common/Size2D.cs

```diff
@@ -127,13 +141,16 @@ public int Width
                 Interop.Vector2.WidthSet(SwigCPtr, (float)value);
                 if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
 
+                userWidth = value;
+                useUserWidth = true;
+
                 callback?.Invoke(value, Height);
             }
             get
             {
                 float ret = Interop.Vector2.WidthGet(SwigCPtr);
                 if (NDalicPINVOKE.SWIGPendingException.Pending) throw new InvalidOperationException("FATAL: get Exception", NDalicPINVOKE.SWIGPendingException.Retrieve());
-                return (int)ret;
+                return (useUserWidth == true) ? userWidth : (int)ret;
```

Comment:
float가 표현하는 int.MaxValue를 int로 변환할때 발생하는 변환 오류이기 때문에 사용자가 저장한 값을 기록하여 리턴하기 보다는 변환하는 방법을 고쳐서 오류를 수정하는것이 다 나을듯합니다.
그래야지 Native에서 설정된 값일지라도 문제 없이 변환이 될듯합니다.

```suggestion
                return ClampToInt(ret);
                
                int ClampToInt(double v) =>
             v > int.MaxValue ? int.MaxValue
            : v < int.MinValue ? int.MinValue
            : (int)v;
```
(꼭, float를 double로 변환 후 사용해야 합니다.)

Reply:
코멘트 감사드립니다.
코멘트주신 것처럼 원래 패치에 문제가 있고 제안해주신 패치 내용이 문제를 근본적으로 해결할 수 있어서 제안해주신대로 패치를 수정하였습니다.
감사합니다.